blueskyacademy/bluesky-nextjs
# add translation

## Description
- Add locales for Model Job and Testimonial

## Why
- User can see lists of jobs and parents' feedback in Vietnamese and English


## How to QA
- User can choose English or Vietnamese to see list of jobs or parents' feedback


## Screenshots
![Screen Shot 2022-01-18 at 21 09 13](https://user-images.githubusercontent.com/74655110/149952922-ae1a387c-c449-4b51-a536-fd8eaaab8797.png)
![Screen Shot 2022-01-18 at 21 09 25](https://user-images.githubusercontent.com/74655110/149952935-06fb99d6-1cda-48a1-8741-c44bcc58297b.png)
